custom_components/waste_collection_schedule/waste_collection_schedule/source/guildford_gov_uk.py

```diff
@@ -0,0 +1,72 @@
+import json
+from datetime import datetime
+
+import requests
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+
+TITLE = "guildford.gov.uk"
+DESCRIPTION = "Source for guildford.gov.uk services for Guildford, UK."
+# Find the UPRN of your address using https://www.findmyaddress.co.uk/search
+URL = "https://guildford.gov.uk"
+TEST_CASES = {
+    "GU12": {"uprn": "10007060305"},
+    "GU1": {"uprn": "100061398158"},
+    "GU2": {"uprn": "100061391831"},
+}
+
+ICONS = {
+    "Refuse": "mdi:trash-can",
+    "Food": "mdi:food-apple",
+    "Recycling": "mdi:recycle",
+    "Garden Waste": "mdi:leaf",
+}
+
+
+class Source:
+    def __init__(self, uprn):
+        self._uprn = uprn
+
+    def fetch(self):
+        # The API uses this framework cookie, which seems to last 2 weeks.
+        framework = "-SjNAdgW9yv96YgKI8MiFA"
+        headers = {'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64; rv:91.0) Gecko/20100101 Firefox/91.0','Accept': '*/*','Accept-Language': 'en-GB,en;q=0.5','Referer': 'https://my.guildford.gov.uk/customers/s/view-bin-collections','X-SFDC-Page-Scope-Id': '2f1b8c8c-7f21-4c8e-97ff-7d382bef472e', 'X-SFDC-Request-Id': '310489300000000d0f' ,'Content-Type': 'application/x-www-form-urlencoded;charset=UTF-8','Origin': 'https://my.guildford.gov.uk', 'Connection': 'keep-alive', 'Cookie': 'renderCtx=%7B%22pageId%22%3A%22d16a45fafd2d-434c-9867-13157cad5ae0%22%2C%22schema%22%3A%22Published%22%2C%22viewType%22%3A%22Published%22%2C%22brandingSetId%22%3A%2291620535-2a5f-4116-830f-f7be831ad395%22%2C%22audienceIds%22%3A%226Au4K000000L1vJ%2C6Au4K000000L1vF%22%7D; CookieConsentPolicy=0:1; LSKey-c$CookieConsentPolicy=0:1; CookieConsentPolicy=0:0; LSKey-c$CookieConsentPolicy=0:0; pctrk=cd6cad4a-1a62-4b4b-a9e7-5df4a5eede9b; sfdc-stream=!av30lCajg42+B19mc4XCkPx8HEy5Mkzqs0SBfqVYJh6EaGzz631DI9Km3o0DTHGHsyz5MSeL5l6yMg==; force-proxy-stream=!LAorEQlyJAsY1yCvxttwW2ftStfhSkP54cqVjoflXoWOf54cRL0W4TSeMTqu5hHUMI5Z1RLqq/fmfQ==; force-stream=!av30lCajg42+B19mc4XCkPx8HEy5Mkzqs0SBfqVYJh6EaGzz631DI9Km3o0DTHGHsyz5MSeL5l6yMg==', 'Sec-Fetch-Dest': 'empty', 'Sec-Fetch-Mode': 'cors','Sec-Fetch-Site': 'same-origin','TE': 'trailers'}
```

Comment:
Add only the mandatory headers.

Reply:
Not sure how to figure out which headers are mandatory ? This is my first time doing this, I just enabled firefox debugging and grabbed the curl command which the page page sends for the search for my postcode.

info.md has been updated, I couldn't find readme.md ?